Vizzuality/heco-invest
# [FE](feat): update priority landscapes [LET-1404]

# Update priority landscapes

## Testing instructions

Check Priority landscapes section on `/for-investors` route:
- Now we should have 9 projects.
- Hovering theme to see description (except 2 of them which dont have description yet).
- Hovering them to see if image change.
- Hovering old ones too to check if there is any "blink" between images.
- Scroll to check if image map is sticky.

## Tracking

[LET-1404](https://vizzuality.atlassian.net/browse/LET-1404)


[LET-1404]: https://vizzuality.atlassian.net/browse/LET-1404?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
